pushapkscript/googleplay.py

```diff
@@ -35,7 +35,8 @@ def craft_push_apk_config(context, apks, google_play_strings_path=None):
         push_apk_config['rollout_percentage'] = payload['rollout_percentage']
 
     # Only known android_products are allowed to connect to Google Play
-    if not is_allowed_to_push_to_google_play(context):
+    # and only if the configuration of the pushapkscript instance allows it
+    if not is_allowed_to_push_to_google_play(context) or context.config.get('do_not_contact_google_play'):
```

Comment:
I wonder if we should get rid of `is_allowed_to_push_to_google_play()` now that we have this config. What do you think?

Reply:
That's a good question, and I'd love to remove `do_not_contact_google_play` from the task payload. However, do we know that it's not currently used that way right now? I don't want to break existing workflows. I want to be sure nobody is doing stuff like: 

* Someone testing checks before pushing to Google Play on the prod `pushapkscript` instance, but not actually wanting anything deployed.
* Maybe , when deploying a new product, rather than not scheduling the `pushapkscript` task (while we wait for the first build to be manually deployed to GP), someone is scheduling this task with `do_not_contact_google_play` :thinking: 

If nobody is using the property on the task, then I'm all for removing it to reduce complexity :)

Reply:
Mmmm, I might be missing something. As far as I remember, `do_not_contact_google_play` isn't exposed to the task definition: https://github.com/mozilla-releng/pushapkscript/blob/32fd2cd81d98e2181b5662eb7c8adc3dcbb3862f/pushapkscript/googleplay.py#L71

Reply:
:thinking: you're absolutely right, whoops. Good point!